style.css

```diff
@@ -316,5 +316,145 @@ a {
     margin: 8px 20px;
   }
 
+  #contact {
+    padding: 80px 0 0 0;
+    background-color: #fff;
+  }
+
+  footer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border-radius: 70px 0 0 0;
+    background-color: #6070ff;
+  }
+
+  footer h2 {
+    margin-top: 103px;
+    margin-bottom: 6px;
+    color: #fff;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  #para {
+    color: #ebebff;
+    margin: 6px 24px;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 28px;
+  }
```

Comment:
- [ ] It would be better to center the text so your design matches the Figma design.
- Your design:
<img width="152" alt="image" src="https://user-images.githubusercontent.com/77190203/155186043-393dc9a8-17d1-466b-8829-2f4190d9daec.png">